### PR TITLE
10 analysis gwas all

### PR DESCRIPTION
**Pull request by anastasia-shch**
- Added UI for submitting GWAS run: first creates connection using `FinnGenUtilsR:: createSandboxAPIConnection` and then submits the run using `FinnGenUtilsR:: runGWASAnalysis` 
- Added server side for submitting GWAS run

### CODE REVIEW

**Review by javier-gracia-tabuenca-tuni — CHANGES_REQUESTED** [review]

File: R/mod_runGWAS.R [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+
+      controlsCohortTableHandler <- r_connectionHandlers$databasesHandlers[[r_data$controlsDatabaseId]]$cohortTableHandler
+
+      controlsCohortData <- HadesExtras::getCohortDataFromCohortTable(
+        connection = controlsCohortTableHandler$connectionHandler$getConnection(),
+        cdmDatabaseSchema = controlsCohortTableHandler$cdmDatabaseSchema,
+        cohortDatabaseSchema = controlsCohortTableHandler$cohortDatabaseSchema,
+        cohortTable = controlsCohortTableHandler$cohortTableNames$cohortTable,
+        cohortNameIds = tibble::tibble(cohortId=r_data$controlsCohortId, cohortName=r_data$controlsCohortName))
+
```
[the remark this answers is not in the document]
Comment (anastasia-shch) [reply R/mod_runGWAS.R:207]: I was thinking about it,  but I was unsure about the database id - is it possible that cases and controls have different database id?

[the remark this answers is not in the document]
Comment (javier-gracia-tabuenca-tuni) [reply R/mod_runGWAS.R:207]: ah, no, they must come from the same

[the remark this answers is not in the document]
Comment (anastasia-shch) [reply R/mod_runGWAS.R:207]: ok, will update it then

[the remark this answers is not in the document]
Comment (javier-gracia-tabuenca-tuni) [reply R/mod_runGWAS.R:207]: you can copy code from codeWAS menu, for example
image

[the remark this answers is not in the document]
Comment (anastasia-shch) [reply R/mod_runGWAS.R:207]: Ok, thanks. Do we have several databases to choose from in SB, or is it always the latest one? For instance right now DF12 is the only option possible for selection in CO2?

File: R/mod_runGWAS.R [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+    #
+    # render cases cohort picker
+    #
+    shiny::observeEvent(input$selectCasesCohort_pickerInput, {
+      shiny::req(r_workbench$cohortsSummaryDatabases)
+      shiny::req(input$selectCasesCohort_pickerInput)
+      shiny::req(input$selectDatabases_pickerInput_gwas)
+
+      browser()
```
Comment (javier-gracia-tabuenca-tuni) [inline R/mod_runGWAS.R:155]: remove this

**Review by javier-gracia-tabuenca-tuni — CHANGES_REQUESTED** [review]
in codewas 
when case cohort is selected
it cannot be selected anymore as controls
could you fix that ?

**Comment by anastasia-shch** [comment]
Updated based on your latest comments. Please, take a look when you have time

**Pull request merged**